components/__tests__/Template-test.js

```diff
@@ -37,7 +37,7 @@ describe('Template', () => {
     renderer.render(<Template {...props} />);
     let out = renderer.getRenderOutput();
 
-    expect(out).toEqualJSX(<div dangerouslySetInnerHTML={{__html: 'it works with strings'}}></div>);
+    expect(out).toEqualJSX(<div className={undefined} dangerouslySetInnerHTML={{__html: 'it works with strings'}}></div>);
```

Comment:
But still, what is the difference between testing `<div className={undefined}>` and testing `<div>`? I would expect a props with a value of `undefined` to just not be in the resulting JSX